component/topsql/query/default_query.go

```diff
@@ -591,22 +592,29 @@ func (dq *DefaultQuery) fillText(name string, sqlGroups *[]sqlGroup, fill func(R
 			for _, series := range group.planSeries {
 				planDigest := series.planDigest
 				var planText string
+				var encodedPlan string
 
 				if len(planDigest) != 0 {
 					r, err := tx.QueryDocument(
-						"SELECT plan_text FROM plan_digest WHERE digest = ?",
+						"SELECT plan_text, encoded_plan FROM plan_digest WHERE digest = ?",
 						planDigest,
 					)
 					if err == nil {
-						_ = document.Scan(r, &planText)
+						_ = document.Scan(r, &planText, &encodedPlan)
 					}
 				}
 
 				planItem := RecordPlanItem{
 					PlanDigest:   planDigest,
-					PlanText:     planText,
 					TimestampSec: series.timestampSecs,
 				}
+
+				if len(planText) > 0 {
+					planItem.PlanText = planText
+				} else if len(encodedPlan) > 0 {
+					planItem.PlanText, _ = plan.Decode(encodedPlan)
```

Comment:
In theory there will be no errors unless the codec is updated incompatible, so do we need to log error here to catch this potential problem?

Reply:
Make sense